DESCRIPTION

```diff
@@ -1,14 +1,19 @@
-Package: admiral.test
+Package: cdiscdata 
```

Comment:
Please remove trailing whitespace.

Reply:
i guess this change no longer needed from the further discussions?